framework/wazuh/tests/test_vulnerability.py

```diff
@@ -58,6 +58,8 @@ def send_msg_to_wdb(msg, raw=False):
     ({'filters': {'version': '0.1.0', 'architecture': 'ARM'}}, 'cve', ['CVE-2019-1020019']),
     ({'filters': {'cve': 'CVE-2019-1020016'}}, 'cve', ['CVE-2019-1020016']),
     ({'filters': {'cve': 'CVE-2019-1020016', 'architecture': 'ARM'}}, 'cve', []),
+    ({'filters': {'status': 'OBSOLETE'}},'cve',[]),
```

Comment:
add more a couple more filters with actual results and not empty cve, based on previously requested changes in schema

Reply:
Arreglado